components/com_content/src/View/Category/FeedView.php

```diff
@@ -50,9 +50,21 @@ protected function reconcileNames($item)
 		$item->description = '';
 		$obj = json_decode($item->images);
 
-		if (!empty($obj->image_intro))
+		// Set feed image to image_intro or if that's empty, to image_fulltext
+		$itemImage = '';
+
+		if (!empty($obj->{'image_intro'}))
+		{
+			$itemImage = $obj->{'image_intro'};
+
+		} elseif (!empty($obj->{'image_fulltext'}))
```

Comment:
```suggestion
		}
		elseif (!empty($obj->{'image_fulltext'}))
```
Fix code style errors reported by drone.